src/stylesheets/main.scss

```diff
@@ -376,4 +377,8 @@ p code {
   pre + h4 {
     padding-top: govuk-spacing(2);
   }
+
+  > :last-child {
```

Comment:
on `/typography` and `/spacing` this reduces the amount of margin at the bottom of the last example but I can't see it having an effect on the back to top link. I might be missing something.

Reply:
These changes to content do not impact the back to top link, they just make sure when you scroll to the bottom that the content is in line with the back to top link.

Reply:
Thanks Nick. I think a code comment to explain that would be great as it's not clear from the code this is what's happening 🙏